### ISSUE TITLE
Paginating Support (consumers endpoint)

### ISSUE THREAD

**Issue by vl4d** [mybuilder/kongfig#30]
While running kongfig, I ran into a problem where I have approx 300+ consumers; which would require to request consumers in increments as there is a limit of 100 on the endpoint at the moment. It results in:

> Error: Content overflow on {kong-api:8001}/consumers, pagination not supported

to be outputted into the output file. 

Is pagination support being considered? I may take a shot at adding it in.

**Comment by JnMik** [comment]
I have not reached the 300+ consumers yet, but I will def. need this in a near future.

So +1 for your effort of adding it in if you do ! :)

**Comment by CyExy** [comment]
@vl4d Sorry for a late response. I can look into implementing something if you have not already started.

Are you managing these 300+ consumers using kong or you are using something else to do that? Basically would it be useful to have a config param that would turn off managing consumers through kongfig?

**Comment by vl4d** [comment]
Thanks  for following up. We would love the optional param to turn off managing consumers through kong. (I hacked around and turned it off myself )

But yes I am managing the 300+ consumers in kong. We were migrating from cassandra to postgres when we noticed the problem. I have not started as yet as we automatically provision consumers in kong so it worked itself out. But the feature would be lovely as well.
